src/components/transactions/TxDetails/Summary/SafeTxHashDataRow/index.tsx

```diff
@@ -4,6 +4,24 @@ import { type SafeTransactionData, type SafeVersion } from '@safe-global/safe-co
 import { getEip712TxTypes } from '@safe-global/protocol-kit/dist/src/utils'
 import useSafeAddress from '@/hooks/useSafeAddress'
 import useChainId from '@/hooks/useChainId'
+import semverSatisfies from 'semver/functions/satisfies'
+
+const NEW_DOMAIN_TYPE_HASH_VERSION = '>=1.3.0'
+
+export function getDomainHash({
+  chainId,
+  safeAddress,
+  safeVersion,
+}: {
+  chainId: string
+  safeAddress: string
+  safeVersion: SafeVersion
+}): string {
+  return TypedDataEncoder.hashDomain({
+    ...(semverSatisfies(safeVersion, NEW_DOMAIN_TYPE_HASH_VERSION) && { chainId }),
```

Comment:
ChatGPT has a point, it would be more readable with
`const includeChainId = semverSatisfies(safeVersion, NEW_DOMAIN_TYPE_HASH_VERSION)`

Reply:
Changed in 0fa8cff887e04e1998333e43c67bd3aa679f4aea.